velox/functions/prestosql/StringFunctions.h

```diff
@@ -132,6 +132,18 @@ struct HmacSha256Function {
   }
 };
 
+/// hmac_sha512(varbinary) -> varbinary
+template <typename T>
+struct HmacSha512Function {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  template <typename TOuput, typename TInput>
+  FOLLY_ALWAYS_INLINE bool
+  call(TOuput& result, const TInput& data, const TInput& key) {
```

Comment:
Can we always return void here instead of boolean. We never return null here? The same for HmacSha256Function? @duanmeng @Yuhta Thanks!

Reply:
I think we can return `void` here, same as `sha512`

Reply:
@xiaoxmeng @Yuhta OK, I've fixed the return type of both hmac_sha256 and hmac_sha512.
BTW, should I also fix the return types of `crc32`, `sha256`, `md5` and `XxHash64`, and if so, in a separate pr or this pr?

Reply:
Yes you can fix them in the same PR as long as they are in the same file.

Reply:
OK, done.